filament/backend/include/backend/DriverEnums.h

```diff
@@ -233,14 +237,114 @@ static inline constexpr bool hasShaderType(ShaderStageFlags flags, ShaderStage t
     }
 }
 
-enum class DescriptorType : uint8_t {
-    UNIFORM_BUFFER,
-    SHADER_STORAGE_BUFFER,
-    SAMPLER,
-    INPUT_ATTACHMENT,
-    SAMPLER_EXTERNAL
+enum class TextureType : uint8_t {
+    FLOAT,
+    INT,
+    UINT,
+    DEPTH,
+    STENCIL,
+    DEPTH_STENCIL
 };
 
+ constexpr std::string_view to_string(TextureType type) noexcept {
```

Comment:
nit
```suggestion
constexpr std::string_view to_string(TextureType type) noexcept {
```
A bunch of `constexpr` functions in this file have a preceding space